docs/VARIABLES.md

```diff
@@ -132,6 +132,46 @@ Default:  true
 
 ***
 
+### logredactor_enabled
+
+Boolean to enable logredactor for all components. Works only when enabled along with custom_log4j. If enabled, ensure correct value of logredactor_rule_path/url
+
+Default:  false
+
+***
+
+### logredactor_rule_path_local
+
+Location of the logredactor rules file on the control node. Ensure that the rules files exist here if logredactor enabled. This'll be copied to logredactor_host_rule_path for all components. Not required if logredactor_rule_url is specified.
+
+Default:  ../../../rules.json
+
+***
+
+### logredactor_rule_path
+
+Full path and name of the rules file on all components. This is the location which will be referenced in the log4j property file on the redactor appender. Not required if logredactor_rule_url is specified. Ensure that the path/directory exists.
+
+Default:  /etc/kafka/rules.json
+
+***
+
+### logredactor_rule_url
+
+URL address that contains or redirects to the redaction rules. If not "", this will be used to fetch rules. logredactor_rule_path will be ignored.
+
+Default:  ""
+
+***
+
+### logredactor_policy_refresh_interval
+
+If present, it's used to specify a time in ms for how often the file system or URL of the policy rules will be checked for changes. Default is "" i.e. policy rules read only at startup. Can be specified as logredactor_policy_refresh_interval: 7000
+
+Default:  ""
```

Comment:
Since its an integer, do we have better default value like -1/0 ?

Reply:
After our discussion, I did look at documentation to see suggestions on this. 
AFAIU, if we don't want to refresh, we have to skip this property. No integer value is equivalent of disabling. 
0 is a valid value as per my test.
-1 is something we can use as default, and then have logic to skip adding the property if value is -1. Currently, the same logic deals with ""

What do you think?

Reply:
If we have to implement this, I would go for 0 as value for "Never refresh". We can add the same in variable description.

Reply:
Yeah, can be done. The logic would be same, skip property if value is 0. Will make this change then